synfig-studio/src/synfigapp/value_desc.h

```diff
@@ -217,6 +245,15 @@ class ValueDesc
 		if (parent_desc) parent_desc->links_count++;
 	}
 
+	// move
+	ValueDesc(ValueDesc&& other) noexcept
```

Comment:
I was just placing everything in a single method to avoid mistakes. For example, someone adds a new "field"/property to `ValueDesc` and forget to add to both move-related methods.

And these methods declarations are only due to this explicit pointer `parent_desc` :/